app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[name] = self
```

Comment:
You do not need to do with self.__class__.people
Two options:
1. self.people
2. Person.people

Reply:
1. self.people is an object class but not object attribute
2. this is bad hardcoding practice